connect-app/app/screens/Home.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { StyleSheet, Text, TouchableOpacity, View } from "react-native";
+import { useNavigation } from "@react-navigation/core";
+import { auth } from "../../FirebaseConfig";
```

Comment:
So, you used Google Firebase as a backend NoSql db? Is this because it already has predefined components?

Reply:
or Libraries

Reply:
I remember the approach we had discussed in brainstorming session reg authentication to go with storing email & encrypted password in the db. Rather than that approach, I feel like going with Firebase as we can have flexibility to add sign in methods with Google, Facebook in the future.

Reply:
okay. Is it possible to add this authentication in the backend? I heard from RK that we never implement auth in UI and auth should be handled in backend code.